pom.xml

```diff
@@ -8,7 +8,7 @@
   <!-- PROJECT NAME -->
   <name>Ewon Flexy Extensions Library</name>
   <!-- PROJECT VERSION -->
-  <version>1.13.10</version>
+  <version>1.14.0</version>
```

Comment:
This is more of a discussion, but what warrants this version being 1.14.0 versus 1.13.11? 

A bump in the major version number seems inappropriate here, as there is only a single bug fix and no new features. (Per the changelog)